services/director-v2/setup.cfg

```diff
@@ -11,3 +11,4 @@ commit_args = --no-verify
 asyncio_mode = auto
 markers =
 	testit: "marks test to run during development"
+  acceptance_test: "marks tests as 'acceptance tests' i.e. does the system do what the user expects? Typically those are workflows."
```

Comment:
careful here with tabs and spaces. ! setup will not understand the marker if you did it wrong

I tried to setup vscode to format *.cfg with tabs/spaces but did not manage. If you find a way, please add it to the .vscode/settings.template.json

Reply:
ok.